saleor/graphql/checkout/mutations/checkout_delivery_method_update.py

```diff
@@ -201,13 +210,18 @@ def _update_delivery_method(
             delete_external_shipping_id(checkout=checkout)
         checkout.shipping_method = shipping_method
         checkout.collection_point = collection_point
+        invalidate_prices_updated_fields = invalidate_checkout_prices(
+            checkout_info, lines, manager, discounts or [], save=False
+        )
         checkout.save(
             update_fields=[
                 "private_metadata",
                 "shipping_method",
                 "collection_point",
+                "price_expiration",
```

Comment:
`"price_expiration"` can be removed from here as it's already in `invalidate_prices_updated_fields`.